de.tudarmstadt.ukp.wikipedia.api/src/test/java/de/tudarmstadt/ukp/wikipedia/api/CategoryTest.java

```diff
@@ -17,21 +17,24 @@
  *******************************************************************************/
 package de.tudarmstadt.ukp.wikipedia.api;
 
-import static org.junit.Assert.assertEquals;
```

Comment:
Please avoid star imports ;)

Reply:
I don't see a * import at this line here, as the comment is on a deletion of a line.